scripts/setup-ubuntu.sh

```diff
@@ -75,14 +91,19 @@ function install_build_prerequisites {
     ninja-build \
     checkinstall \
     git \
+    pkg-config \
+    cmake-data \
```

Comment:
nit:)Do we actually need cmake-data when installing via pip?

Reply:
Good point. We don't install cmake from the system. When looking for PkgConfig I though we might need more stuff. But this doesn't make sense. I'm trying it out without this package.